unittests/resource_limits_test.cpp

```diff
@@ -136,33 +136,37 @@ BOOST_AUTO_TEST_SUITE(resource_limits_test)
    } FC_LOG_AND_RETHROW();
 
    /**
-    * create 5 accounts with different weights, verify that the capacities are as expected and that usage properly enforces them
+    * create 12 accounts with different weights, verify that the capacities are as expected and that usage properly enforces them
     */
-
-
-// TODO: restore weighted capacity cpu tests
-#if 0
    BOOST_FIXTURE_TEST_CASE(weighted_capacity_cpu, resource_limits_fixture) try {
-      const vector<int64_t> weights = { 234, 511, 672, 800, 1213 };
-      const int64_t total = std::accumulate(std::begin(weights), std::end(weights), 0LL);
+      const vector<int64_t> weights = { 1, 3, 9, 234, 511, 672, 800, 1213, 4242, 77777, 888888, 999999 };
+      const uint128_t total = std::accumulate(std::begin(weights), std::end(weights), 0LL);
       vector<int64_t> expected_limits;
-      std::transform(std::begin(weights), std::end(weights), std::back_inserter(expected_limits), [total](const auto& v){ return v * config::default_max_block_cpu_usage / total; });
+      std::transform(std::begin(weights), std::end(weights), std::back_inserter(expected_limits), [total](const auto& v){
+         uint128_t value = v;
+         uint128_t account_cpu_usage_average_window = config::account_cpu_usage_average_window_ms / config::block_interval_ms;
```

Comment:
Just a remark, `account_cpu_usage_average_window_ms` is a very un-intuitive name for what it really is (`milliseconds_per_day`)

Also, shouldn't  `account_cpu_usage_average_window` be better named `num_slots_per_day`?

Reply:
`account_cpu_usage_average_window_ms` doesn't have to be milliseconds per day.
Renamed `account_cpu_usage_average_window`  to `account_cpu_usage_window `